### ISSUE TITLE
Sentry 5.7.2 with timber integration crashes the app when formats a message with args

### ISSUE THREAD

**Issue by sdex** [getsentry/sentry-java#1985]
### Integration

sentry-android-timber

### Build System

Gradle

### AGP Version

7.1.3

### Proguard

Disabled

### Version

5.7.2

### Steps to Reproduce

1. Add the log:
`Timber.d("test %s, %s", 1, 1);`
2. The app crashes.

Version 5.7.1 works fine. It seems like the #1974 PR causes this. The `args` is passed incorrectly - missing asterisk. 

### Expected Result

A log message:
`test 1 1`

### Actual Result

```
Caused by: java.util.MissingFormatArgumentException: Format specifier '%s'
            	at java.util.Formatter.format(Formatter.java:2522)
            	at java.util.Formatter.format(Formatter.java:2458)
            	at java.lang.String.format(String.java:2842)
            	at timber.log.Timber$Tree.formatMessage(Timber.kt:172)
            	at timber.log.Timber$Tree.prepareLog(Timber.kt:161)
            	at timber.log.Timber$Tree.d(Timber.kt:51)
            	at io.sentry.android.timber.SentryTimberTree.d(SentryTimberTree.kt:60)
            	at timber.log.Timber$Forest.d(Timber.kt:292)
            	at timber.log.Timber.d(Unknown Source:2)
            	at the.app.MainActivity.onCreate(MainActivity.java:109)
```

**Comment by romtsn** [comment]
Indeed the asterisk is the problem, sorry, this slipped through the review 🙈 Will roll out a fix for that shortly, thanks for reporting!

**Comment by romtsn** [comment]
@sdex we just released 5.7.3 with this fix, please give it a try. Thanks!

**Comment by sdex** [comment]
@romtsn the new version works fine. Thanks for the quick fix!
